### ISSUE TITLE
Select skill for fine-tune

### ISSUE THREAD

**Issue by linyuhongg** [rll-research/cic#2]
Hi,

The paper states that during finetune the skill is searched through grid-search. However, in the code a fixed skill used, is that normal?

**Comment by amberxie88** [comment]
The grid search can/should be done in a separate file. The finetune file just finetunes the fixed skill

**Comment by seolhokim** [comment]
Actually, not fixed. It change every 50 time steps in this repo. Rather, I wonder how the grid sweep was executed.

**Comment by sicko-mode-df** [comment]
> 

Hi, have you got clear with this problem? I'm confused with this issue too.

**Comment by amberxie88** [comment]
Finetuning skill selection is not implemented in the CIC repository. However, we build off of the CIC repository and implement skill selection in Intrinsic Reward Matching (IRM): https://github.com/ademiadeniji/irm/blob/main/finetune.py#L303 

Feel free to take a look for an example

**Comment by sicko-mode-df** [comment]
> 

Thank you very much for your help, have a nice day!
